<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="WPUSM24398A" name="com.deere.testCases.ClaimReports" tests="1" failures="0" timestamp="13 Oct 2017 07:00:37 GMT" time="74.939" errors="1">
  <testcase name="test_LaborStatsReport" time="74.939" classname="com.deere.testCases.ClaimReports">
    <error type="java.lang.AssertionError" message="File is not downloaded expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: File is not downloaded expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.deere.global.CommonFunctions.assertTrue(CommonFunctions.java:1014)
	at com.deere.testCases.ClaimReports.test_LaborStatsReport(ClaimReports.java:299)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </error>
  </testcase> <!-- test_LaborStatsReport -->
</testsuite> <!-- com.deere.testCases.ClaimReports -->
